dom/nodes/Document-createElement-namespace.html

```diff
@@ -39,15 +43,15 @@
 }, "Created element's namespace in created XML document");
 test(function() {
   testDoc(document.implementation.createDocument("http://www.w3.org/1999/xhtml", "html", null),
-          "application/xml");
+          "application/xhtml+xml");
```

Comment:
Correct, as noted in the commit message.  I expect it will be there within a couple of days, and didn't want to bother updating the test twice.  If it actually gets rejected, I'll fix the test then.